packages/core/src/configs/node/jest.setup.js

```diff
@@ -50,8 +51,12 @@ const removeScenario = async (scenario) => {
     // get unique model names only
     models = Array.from(new Set(models))
 
-    for (const model of models) {
-      await db.$queryRaw(`DELETE FROM ${model}`)
+    const prismaModelNames = (
```

Comment:
We are now running this for every scenario, would it make sense to instead of attaching this to if a scenario is present, run this every time a test has ran? So basically make it a poor mans transaction?

Reply:
Great catch -- I hadn't realized it would miss cases where we weren't using scenarios, but _were_ adding records to the database.  I believe that case is now taken care of, and I've renamed the function to reflect this.